micro_g/scripts/pose_servoing_controller.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/python3
```

Comment:
Might be good to move this to a separate package. The meta package is usually used to manage building the project (so that you can use package-select a little more freely). The canonical name would be something like `micro_g_control` or `micro_g_controllers`. Usually the launch files are in their own package too, but I've found that to be a bit redundant